cdap-security/src/main/java/co/cask/cdap/security/store/SecureStoreUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.security.store;
+
+import co.cask.cdap.common.conf.CConfiguration;
+import co.cask.cdap.common.conf.Constants;
+
+/**
+ *
```

Comment:
That is inside the kerberos package and only has kerberos related methods. Ok to expand that?


> On Jul 20, 2016, at 12:14 AM, Terence Yim <notifications@github.com> wrote:
> 
> In cdap-security/src/main/java/co/cask/cdap/security/store/SecureStoreUtils.java <https://github.com/caskdata/cdap/pull/6066#discussion_r71475579>:
> 
> > + * http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
> > + * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
> > + * License for the specific language governing permissions and limitations under
> > + * the License.
> > + */
> > +
> > +package co.cask.cdap.security.store;
> > +
> > +import co.cask.cdap.common.conf.CConfiguration;
> > +import co.cask.cdap.common.conf.Constants;
> > +
> > +/**
> > + *
> BTW, there is already a SecurityUtil class. Probably better merge with that one ( @anwar6953 <https://github.com/anwar6953> PR has some changes to it as well).
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/caskdata/cdap/pull/6066/files/07b78a6b2dd10e0a6cac763002474c39732b418c#r71475579>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAFGk1iHp-cW-sbx8VNzI7SMl2vAzLP_ks5qXcrCgaJpZM4JOEs0>.
>